policyengine_us/parameters/gov/states/az/tax/income/subtractions/military_pension/cap.yaml

```
@@ -0,0 +1,16 @@
+description: Arizona excludes benefits, annuities and pensions from uniformed services of the United States of up to this amount.
```

To clarify that it's not all military benefits, or all pensions among military retirees.
```suggestion
description: Arizona subtracts from gross income any benefits, annuities and pensions as retired or retainer pay of the uniformed services of the United States, up to this amount.
```